deployer/run.sh

```diff
@@ -169,9 +169,13 @@ keytool -noprompt -import -v -trustcacerts -alias $hawkular_cassandra_alias -fil
 
 echo "Importing the CA Certificate into the Cassandra Truststore"
 keytool -noprompt -import -v -trustcacerts -alias ca -file ${dir}/ca.crt -keystore $dir/hawkular-cassandra.truststore -trustcacerts -storepass $hawkular_cassandra_truststore_password
+keytool -noprompt -import -v -trustcacerts -alias metricca -file ${dir}/hawkular-metrics-ca.cert -keystore $dir/hawkular-cassandra.truststore -trustcacerts -storepass $hawkular_cassandra_truststore_password
+keytool -noprompt -import -v -trustcacerts -alias cassandraca -file ${dir}/hawkular-cassandra-ca.cert -keystore $dir/hawkular-cassandra.truststore -trustcacerts -storepass $hawkular_cassandra_truststore_password
 
 echo "Importing the CA Certificate into the Hawkular Metrics Truststore"
 keytool -noprompt -import -v -trustcacerts -alias ca -file ${dir}/ca.crt -keystore $dir/hawkular-metrics.truststore -trustcacerts -storepass $hawkular_metrics_truststore_password
+keytool -noprompt -import -v -trustcacerts -alias metricsca -file ${dir}/hawkular-metrics-ca.cert -keystore $dir/hawkular-metrics.truststore -trustcacerts -storepass $hawkular_metrics_truststore_password
```

Comment:
It seems weird to not use the cluster CA to issue a certificate & mount that in pods rather than creating your own temporary CA for each deploy, which effectively means noone can ever verify the hawkular server certificate.

@liggitt Any comment on this? Do we have a strategy in place for this kind of thing yet?